app/src/main/kotlin/com/exallium/todoapp/allnotes/AllNotesPresenter.kt

```diff
@@ -28,6 +30,27 @@ class AllNotesPresenter(view: AllNotesView,
 
         adapter.noteClicks().subscribe(showNoteSubscriberFn).addToComposite()
         view.addNoteClicks().map { null }.subscribe(showCreateNoteSubscriberFn).addToComposite()
+
+        setupDeleteNoteSubscription()
+    }
+
+    fun setupDeleteNoteSubscription() {
+        adapter.noteSwipes()
+                .flatMap { model.deleteNote(it).toObservable() }
+                .subscribe(object : Subscriber<Unit>() {
```

Comment:
consider using subscribe({}, {}) method instead of creating this annonymous object